docs/guides/quality.md

```diff
@@ -1,142 +1,212 @@
+
+The official BDMV is an encode of the original source master. Subsequently, it often suffers from various issues (e.g. aliasing, banding, blocking, noise) originating due to a variety of reasons. For example, a lot of anime is natively 720p and is upscaled to 1080p for the Blu-ray. *Even if issues aren't present in the BDMV, they can appear in an encode due to compression.*
+
+These issues can be fixed through filtering, a step which comes before encoding. The video is filtered with tools like [VapourSynth](https://github.com/vapoursynth/vapoursynth) before passing it on to the encoder. Since encoding is a lossy process, filtering is necessary to improve the video quality along with transparent encoder settings. *As a result, any unfiltered encodes are by definition worse than the source.*
+
+See the links below for more information:
 
 - [Examples of problems](http://bakashots.me/guide/index.php)
 - [Encoding guides and more examples](https://guide.encode.moe/encoding/video-artifacts.html)
 - [Advanced encoding guide](https://silentaperture.gitlab.io/mdbook-guide/introduction.html)
 - [Mini encodes and audio](https://kokomins.wordpress.com/2019/10/10/anime-encoding-guide-for-x265-and-why-to-never-use-flac/)
 
-### Types of releases
+!!!warning
+The encoding guides above should give you a general idea of the workflow, *but do note they are heavily outdated and incomplete.*
+!!!
+
+### Types of Releases
+
++++ BDMV
+
+Also known as: [!badge variant="info" text="BDMV" margin="0 8 0 0"]
+[!badge variant="info" text="Blu-ray"]
+
+A complete copy of the original Blu-ray, including menus, extras, adverts, etc. Used as a source for making other releases or encodes. *These are not practical for watching.*
+
+Japanese Blu-rays often have better quality than other countries as they allocate far more bitrate to the video. This is not necessarily intentional, but a side effect of having fewer episodes per disc and not including dub tracks, in contrast to other regions which tend to cram more episodes onto a single disc and have large 5.1 dubs.
+
+There are cases where regional discs do offer better quality. This generally applies when Japanese discs have authoring issues, resulting in the alternatives being better by comparison. For instance, Italian discs from Dynit frequently offer the best video even at lower bitrates due to good filtering and better-optimized encode settings.
+
+These can be mainly found on [U2](https://u2.dmhy.org) (Private), however [Nyaa](https://nyaa.si) (Public) and [Skyeysnow](https://skyeysnow.com) (Private, Open Signup) will have some too. *Release groups don't matter for BDMVs because they are 1:1 copies of the disc and should instead be chosen based on region.*
+
++++ BD Remux
+
+Also known as: [!badge variant="info" text="BD Remux" margin="0 8 0 0"]
```

Comment:
also known as just "remux"

Reply:
Resolved in https://github.com/Snaacky/thewiki/pull/86/commits/e17de4310044aa9fdd9511c35a0ce45da06b2def.